templates/vue-demo-store/nuxt.config.ts

```diff
@@ -3,6 +3,9 @@ import { VueDisableInputsBeforeMount } from "vite-vue-plugin-disable-inputs";
 // https://v3.nuxtjs.org/docs/directory-structure/nuxt.config
 export default defineNuxtConfig({
   runtimeConfig: {
+    shopware: {
+      shopwareEndpoint: "",
+    },
```

Comment:
let's not add this to template as it's only an edge case when someone will use it, instead this "example config" can be moved to documentation as well
```suggestion
```

Reply:
Without this, env will not work and may be confusing. It is easier to just work on env without changing the config file.

Reply:
okay I understand, so at least comment with a link to docs section explaining why there are two shopwareEndpoint settings in the config; this would be confusing for me if I'd fetch this template for the first time :)